src/app.py

```diff
@@ -29,6 +31,12 @@ def get_prediction():
     data = request.get_json()
 
     input_text = data['query']
+    print(input_text)
+
+    label = "placeholder"
```

Comment:
statt einen placeholder zu setzen, kannst du das feld einfach freilassen (in mysql sollte es als required=false eingestellt sein)